### DISCUSSION TITLE
Ignite Expo managed/bare workflow

### DISCUSSION THREAD

**Discussion by saadazizgc** [infinitered/ignite#2447] [Q&A] [answered]
Hi!

If I start a fresh ignite app, the ios and android directories are already generated. 

Does that mean the default ignite app generated is a bare workflow?

I read through this doc:
https://github.com/infinitered/ignite/blob/master/docs/Expo-and-Ignite.md

It says:

> If you still would rather not use Expo Go / CLI, then Ignite is already ready for you -- no "ejecting" necessary.

This implied the out-of-the-box ignite app is a bare workflow, and that, before building the app, one would have to manually `prebuild` the app if they wish to build the app with `eas`?

**Comment by frankcalise** [comment, marked as answer]
Hi @saadazizgc - yes out-of-the-box you can either run via Expo Go (yarn expo:* commands) or via the bare workflow (yarn ios/android). If choosing to build via `eas`, you may have to run `prebuild` and make some config adjustments. Are you looking to just run via Expo?

If so, you can look to this [recipe on the Ignite Cookbook](https://ignitecookbook.com/docs/recipes/PristineExpoProject/) and just abandon the native directories. We'll be clearing this up in a future release

**Reply by saadazizgc** [reply]
Hey Frank, right I see  -  just to ensure I understood you correctly, if I create a project with `npx ignite-cli@latest new PizzaApp`, I will get a bare workflow, with the `ios` and `android` directories. 

By following the link you sent, I can transition to a 'Pristine Expo Project'?

I am looking to run via. expo, and build the app with `eas`, yep.

**Reply by frankcalise** [reply]
Yeah I'd follow that link then and it'll transition you to full expo experience and you can run `eas build:configure` to get your project configured for EAS Build.

The early parts of the [EAS Update](https://ignitecookbook.com/docs/recipes/EASUpdate) how-to also cover that.
